learn.html

```diff
@@ -7,12 +7,65 @@
 ---
 <h1><a id="what"></a>What is a tree-sequence</h1>
 <div>
-    <p>A <em>tree sequence</em> represents the relationships between a set of DNA
-    sequences, and is used for efficient storage and analysis of genetic data.</p>
-    <p>Tree sequences can be created by <a href="software#simulate">simulation</a> or by
-    <a href="software#infer">inferring relationships from genetic variation</a>.
-    In both cases, the efficiency provided by the tree sequence format makes it
-    possible to deal with millions of whole genomes at a time.</p>
+    <p>A <em>succinct tree sequence</em>, or tree sequence for short, represents the
+    relationships between a set of DNA sequences. Tree sequences can be used to store
+    genetic data efficiently, and enable powerful and rapid analysis of millions of
+    whole genomes at a time. They can be created by
+    <a href="software#simulate">simulation</a> or by <a href="software#infer">inferring
+    relationships from genetic variation</a>.</p>
+    
+    The key things a tree sequence can provide are:
+    <dl>
+        <dt>A record of full genetic ancestry</dt>
+        <dd>Tree sequences efficiently store the full genetic history of a set of genomes, by sharing
+        <em>edges</em> (branches) between adjacent trees. <a href="#ancestry">Read more ...</a></dd>
+        <dt>An encoding of DNA data</dt>
+        <dd>Placing mutations on a tree sequence allows lossless representation and compression of
+    DNA datasets. <a href="#mutations">Read more ...</a></dd>
+        <dt>An efficient analysis framework</dt>
+        <dd>Analyses of genetic variation can be reformulated as measurements on trees;
+    applying this to trees with shared structure can (massively?) speed up genome-wide statistical calculations. <a href="#analysis">Read more ...</a></dd>
+    </dl>
+    
+    <h3 id="ancestry">A record of full genetic ancestry</h3>
+    <p>Two basic principles underpin the tree sequence concept:
+    <ol>
+        <li>At a given position along a chromosome, the relationship between a set of
+        genomes can be represented by a single evolutionary tree.</li>
```

Comment:
delete "evolutionary"

Reply:
Why? I put it in there to made it completely clear that we aren't talking about a plant. Is "evolutionary" wrong? I could say "gene tree" but that's confusing to a molecular biologist.

Reply:
The trees don't have anything to do with evolution, do they?